nplbam/app/new_animal.py

```diff
@@ -1,10 +1,24 @@
 from flask import render_template, Blueprint, session as flask_session, redirect
```

Comment:
Note to self for later: Go through and add docstrings to pretty much every function.